FAIMS/FAIMS3
# Update Android Deployment

# Update and Parameterise Android Deployment

## JIRA Ticket

- [BSS-405](https://jira.csiro.com/browse/BSS-405)
- [BSS-344](https://jira.csiro.com/browse/BSS-344)

## Description

To allow us to depoy the BSS app we need to be able to configure the build and
deploy process to make it more flexible.   


## Proposed Changes

- Refactors the Android build and deployment pipeline to add ENV variables that allow us to customse the application deployed to Google Play
- Documents the Android deployment process for future reference
- Removes some old deployment related code that is no longer relevant

## How to Test

We can test by running the Nightly Android Test Build workflow on Github. This should
work as before on the main FAIMS repo and with the right configuration, publish to
the new BSS app from the BSS-NBIC repo.


## Checklist
